src/test/ui/generic-associated-types/issue-80626.stderr

```diff
@@ -0,0 +1,27 @@
+error[E0275]: overflow evaluating the requirement `<A as Allocator>::Allocated<LinkedList<A>>: Sized`
+  --> $DIR/issue-80626.rs:11:14
+   |
+LL |     Next(u8, A::Allocated<Self>),
+   |              ^^^^^^^^^^^^^^^^^^
+   |
+   = note: detected recursive derive for `Sized` in a generic associated type
+help: consider relaxing the implicit `Sized` restriction
+   |
+LL |     type Allocated<T: ?Sized>;
+   |                     ^^^^^^^^
+
```

Comment:
On one hand...yes, this is a suggestion that should work. On the other, is this a suggestion that is normally helpful?